docs/site/LB3-vs-LB4-booting.md

```diff
@@ -0,0 +1,128 @@
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/LB3-vs-LB4-booting.html
+---
+
+## Differences between LoopBack 3 and LoopBack 4 booting process
+
+The booting process in LoopBack 3 is handled by
+[loopback-boot](https://github.com/strongloop/loopback-boot),
+which performs the following tasks:
+
+### In LoopBack 3
+
+#### 1. Configuration of application settings
+
+It determines the execution environment, the application host, the application
+port, the application root directory and other artifact directories.
+
+Then it configures the LoopBack Application object from the `config.json` file
+in the application root directory.
+
+#### 2. Configuration of datasources
+
+Datasources are determined from application options or datasource files. They
+are then attached to the LoopBack Application object.
+
+#### 3. Definition of models
+
+Models are determined from application options or model files,
+[mixins](https://loopback.io/doc/en/lb3/Defining-mixins.html) are applied,
+and they are attached to their respective datasources. They are then attached
+to the LoopBack Application object.
+
+#### 4. Setting up middleware
+
+Middleware from the middleware configuration file are resolved and added to the
+LoopBack 3 middleware chain.
+
+#### 5. Setting up components
+
+Components from the component configuration file are resolved and are added to
+the LoopBack 3 middleware chain.
+
+#### 6. Running boot scripts
+
+Boot scripts are resolved from the `server/boot` directory and run.
+
+### In LoopBack 4
+
+In LoopBack 4, the booting tasks are shared between the
+[RestApplication](./apidocs/apidocs.rest.restapplication.html) class
+and the
+[@loppback/boot](./apidocs/apidocs.boot.html) package. Since your application
+will extend `BootMixin` and `RestApplication`, their separation will not be
+apparent to you, and understanding of how they work together as one as not
+mandatory for developing LoopBack 4 projects.
+
+The following are the list of tasks that are performed in the LoopBack 3
+booting process, and their equivalent in LoopBack 4.
+
+### 1. Configuration of application settings
+
+There is no `config.json` in LoopBack 4. The application options are passed in
+the constructor of the
+[RestApplication](./apidocs/apidocs.rest.restapplication.html) class.
+
+### 2. Configuration of datasources
+
+Datasources in LoopBack 3 are defined in a single `datasources.json` file. In
+LoopBack 4, datasources are defined as TypeScript classes that come with an
+associated JSON configuration file. They all reside in the `src/datasources`
+directory.
+
+`BootMixin` resolves the datasources, and attaches them to the application.
+
+Read more about LoopBack 4 datasource [here](./DataSources.md).
+
+### 3. Definition of models
+
+Although REST APIs are built around Models, they are not a part of the booting
+process in LoopBack 4; Controllers and Repositories are.
+
+Controllers and Repositories import models as regular TypeScript classes.
+
```

Comment:
Can you please briefly explain why we are not booting models?

The main reason is that we need Models at compile time, because as you have correctly pointed out, controllers and repositories are importing them directly.